src/components/TeacherLineItem/TeacherLineItem.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from "./TeacherLineItem.module.css";
+import type { TeacherLineItemProps } from "./TeacherLineItem.definitions";
+
+const TeacherLineItem = ({ teacher }: TeacherLineItemProps) => (
+  <div className={styles["line-item-container"]}>
+    <div className={styles["line-item-container-left"]}>
+      <div className={styles["line-item-name-icon"]} />
+      <div className={styles["line-item-name"]}>
+        {`${teacher.firstName} ${teacher.lastName}`}
+      </div>
+    </div>
+    <div className={styles["line-item-container-right"]} />
+  </div>
+);
+
+export default TeacherLineItem;
```

Comment:
Let's make some visual updates here and give each teacher a grey icon to the left of their name. Looks sort of empty without it.

Reply:
I was thinking when we add editing here thats where it would go.

Reply:
Good idea, lets leave it then